src/App.js

```diff
@@ -1,25 +1,54 @@
-import logo from './logo.svg';
+import React, {useEffect, useState} from 'react';
+import Recipe from './Recipe'
 import './App.css';
 
-function App() {
-  return (
+const App = () => {
+
+  const APP_ID = 'b26f7b04';
```

Comment:
Typically, we want to put variables like APP_ID and APP_KEY in a .env file so they're not publicly exposed (anyone can view our repos and we don't want them to use these values/API keys). Just for future reference!